samples/metrics_monitor/src/cttmetrics_i915_pmu.cpp

```diff
@@ -94,32 +95,91 @@ perf_event_open(struct perf_event_attr *attr,
     return syscall(__NR_perf_event_open, attr, pid, cpu, group_fd, flags);
 }
 
-static uint64_t i915_type_id(void)
+static char *bus_address(int i915, char *path, int pathlen)
 {
-    char buf[1024];
-    int fd, n;
+    struct stat st;
```

Comment:
that's .cpp file. Maybe we will move definitions closer to where we actually use them?